e2e/generateReferences.ts

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
+const puppeteer = require('puppeteer')
+const fs = require('fs');
+
+const main = async () => {
+  const browser = await puppeteer.launch()
+  const page = await browser.newPage()
+  await page.setViewport({
+    height: 200,
+    width: 200,
```

Comment:
Maybe we should get those values from environment variables, or we could specify them as constant values somewhere.

Reply:
It will be hard to share this value between generateReferences and GeneralSvgRenderingTest, also we don't expect them to change any time soon (it would require regenerating all references).